package.json

```diff
@@ -27,7 +27,7 @@
   },
   "homepage": "http://excelbuilderjs.com/",
   "devDependencies": {
-    "@vercel/ncc": "^0.24.0"
+    "@vercel/ncc": "^0.23.0"
```

Comment:
@3cp  isn't this still going to get latest `minor` and so `0.24.0`? I think we should remove the caret or use `~0.23.0` instead

Reply:
Semver has special rule for major version 0, aka pre production.
When major is 0, it treats minor versions (instead normally majors) as breaking change. `"^0.23.0"` would never match 0.24.x.

Reply:
oh ok I didn't know this special rule, thanks for clarifying, I'll merge as is then, it's safe anyway since you already provided the compiled file. Thanks